doc/api-extensions.md

```diff
@@ -247,3 +247,11 @@ around. However, this also entails less efficient storage usage for the
 affected pool.
 The default value for this property is true, i.e. space-efficient snapshots
 will be used unless explicitly set to "false".
+
+## unix-block/unix-char\_device\_rename
```

Comment:
Lets call that "unix_device_rename", that'd be less likely to break random parsers than having a "/" in there :)